enterprise/frontend/src/embedding-sdk/errors/jwt.ts

```diff
@@ -0,0 +1,81 @@
+import { MetabaseError } from "./base";
+
+export function INVALID_SESSION_OBJECT(params: {
+  expected?: string;
+  actual?: string;
+}) {
+  return new MetabaseError(
+    "INVALID_SESSION_OBJECT",
+    `Session object invalid. Expected: ${params.expected}, Actual: ${params.actual}`,
+    params,
+  );
+}
+
+export function INVALID_SESSION_SCHEMA(params: {
+  expected?: string;
+  actual?: string;
+}) {
+  return new MetabaseError(
+    "INVALID_SESSION_SCHEMA",
+    `Session schema invalid. Expected: ${params.expected}, Actual: ${params.actual}`,
+    params,
+  );
+}
+
+export function JWT_ENDPOINT_NON_OK(params: { status: string }) {
+  return new MetabaseError(
+    "JWT_ENDPOINT_NON_OK",
+    `Failed to fetch the session from JWT endpoint, HTTP status: ${params.status}`,
+    params,
+  );
+}
+
+export function BACKEND_ERROR_STATUS(params: {
```

Comment:
[nitpick] I think naming this `REFRESH_TOKEN_BACKEND_ERROR` would make it more specific as the error message already says `Backend returned an error when refreshing the token.`